desktop/core/src/desktop/js/jquery/plugins/jquery.filechooser.js

```diff
@@ -806,48 +806,128 @@ Plugin.prototype.navigateTo = function (path) {
 let num_of_pending_uploads = 0;
 
 function initUploader(path, _parent, el, labels) {
-  new qq.FileUploader({
-    element: el[0],
-    action: '/filebrowser/upload/file',
-    params: {
-      dest: path,
-      fileFieldLabel: 'hdfs_file'
-    },
-    onComplete: function (id, fileName, responseJSON) {
-      num_of_pending_uploads--;
-      if (responseJSON.status == -1) {
-        huePubSub.publish(GLOBAL_ERROR_TOPIC, { message: responseJSON.data });
-      } else if (!num_of_pending_uploads) {
-        _parent.navigateTo(path);
-        huePubSub.publish('assist.' + getFs(getScheme(path)) + '.refresh');
-      }
-    },
-    onSubmit: function (id, fileName) {
-      num_of_pending_uploads++;
-      window.hueAnalytics.log('filechooser', 'uploading-file');
-    },
-    template:
-      '<div class="qq-uploader">' +
-      '<div class="qq-upload-drop-area"><span></span></div>' +
-      '<div class="qq-upload-button">' +
-      labels.UPLOAD_FILE +
-      '</div><br>' +
-      '<ul class="qq-upload-list"></ul>' +
-      '</div>',
-    fileTemplate:
-      '<li>' +
-      '<span class="qq-upload-file"></span>' +
-      '<span class="qq-upload-spinner"></span>' +
-      '<span class="qq-upload-size"></span>' +
-      '<a class="qq-upload-cancel" href="#">' +
-      labels.CANCEL +
-      '</a>' +
-      '<span class="qq-upload-failed-text">' +
-      labels.FAILED +
-      '</span>' +
-      '</li>',
-    debug: false
-  });
+  let uploader;
+  if (window.getLastKnownConfig().hue_config.enable_chunked_file_uploader) {
+    const action = '/filebrowser/upload/chunks/';
+    const qqTemplate = document.createElement('div');
+    qqTemplate.id = 'qq-template';
+    qqTemplate.innerHTML = `
+      <div class="qq-uploader-selector" style="margin-left: 10px; display: flex; flex-direction: column;">
+        <div class="qq-upload-drop-area-selector" qq-hide-dropzone>
+          <span>${'Drop the files here to upload'}</span>
```

Comment:
If you are not using i18n there is no need to use `${}` here. This is just placing a string literal into a string template.